### PR TITLE
[DOCS] Updated integrated support policy for 1.0

### PR DESCRIPTION
**Pull request by joshzzheng**
- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: docs/docusaurus/docs/application_integration_support.md [hunk 1]
```diff
@@ -2,98 +2,69 @@
...
-| Orchestrator         | Airflow ⁵   | Airflow ⁵                                                  |
-
-
-¹ Support for BigQuery will be available in a future release.<br/>
-² Connection strings for various SQL Data Sources are supported, but dialect-specific SQL commands are not.<br/>
-³ Support for Zapier will be available in a future release.<br/>
-⁴ When used with the GX OSS library.<br/>
-⁵ Although only Airflow is supported, GX Cloud and GX OSS should work with any orchestrator that executes Python code.
+¹ We've also seen GX work with the following data sources in the past but we can't guarentee ongoing compatibility. These data sources include Clickhouse, Vertica, Dremio, Teradata, Athena, EMR Spark, AWS Glue, Microsoft Fabric, Trino, Pandas on (S3, GCS, Azure), Databricks (Spark), and Spark on (S3, GCS, Azure).<br/>
+² Support for BigQuery in GX Cloud will be available in a future release.<br/>
```
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/application_integration_support.md:32]: This footnote doesn't appear to be in use in the above table.  You may be missing BigQuery in the above table, or need to remove this footnote and renumber the others.

File: docs/docusaurus/docs/application_integration_support.md [hunk 2]
```diff
@@ -2,98 +2,69 @@
...
+³ This includes configuration storage for Expectations, Checkpoints, Validation Definitions, and Validation Result<br/>
+⁴ config_variables.yml<br/>
+⁵ Although only Airflow is supported, GX Cloud and GX OSS should work with any orchestrator that executes Python code.<br/>
 
 ### GX components
 
 The following table defines the GX components supported by GX Cloud and GX OSS.
```
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/application_integration_support.md]: Throughout the document there are references to GX OSS, but in all of the tables it has been updated to GX Core.  These should be consistent -- "GX OSS" should be replaced with "GX Core" in the text as well as in the table headers.

File: docs/docusaurus/docs/application_integration_support.md [hunk 3]
```diff
@@ -2,98 +2,69 @@
...
+| ------------ | ----------------------------------------------------------------------------------------------- | --------------------------------------------------------------------- | -------------------------------------------------------------------------- |
+| Expectations | See [Available Expectations](/cloud/expectations/manage_expectations.md#available-expectations) | See [Expectations Gallery](https://greatexpectations.io/expectations) | See [Legacy Gallery](https://greatexpectations.io/legacy/v1/expectations/) |
+| GX Agent     | All versions                                                                                    | N/A                                                                   | N/A                                                                        |
 
-## Community-supported
+### Operating systems
 
-The following integrated applications, operating systems, and programming languages are supported by the community.
+The following table defines the operating systems supported by GX Cloud and GX OSS.
```
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/application_integration_support.md]: As noted before: GX OSS -> GX Core.  You should use find/replace to make sure all of these instances get updated.

**Review by Rachel-Reverie — APPROVED** [review]
Two issues to correct: one is a footnote that may need to be removed, the other is the use of "GX OSS" in the text surrounding the tables but "GX Core" in the table headers.

Otherwise, everything LGTM.  I'm marking this approved so that you aren't blocked from merging once you make the recommended changes.

File: docs/docusaurus/docs/application_integration_support.md (hunk 1 above)
Comment (joshzzheng) [reply docs/docusaurus/docs/application_integration_support.md:32]: updated

File: docs/docusaurus/docs/application_integration_support.md (hunk 2 above)
Comment (joshzzheng) [reply docs/docusaurus/docs/application_integration_support.md]: Good catch. Updated all instances of GX OSS with GX Core.

**Pull request merged**